sdk/keyvault/azure-security-keyvault-common/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 4.0.0-beta.4 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Key Bugs Fixed
+
+### Fixed
+
+
```

Comment:
Same question as before: https://github.com/Azure/azure-sdk-for-cpp/pull/2431#discussion_r647856617

@vhvb1989 do you know what has changed that is now bringing in these new sections?

Reply:
No idea, we can ask @danieljurek  in the stand up meeting :) .  I think we can remove it later if not used.  

Maybe it was added after the fact that the Release pipeline will fail if there are no release notes at all. So, this will ensure that if there are not changes, at least the tittles will make the pipeline pass :)